src/check-suite-complete/index.ts

```diff
@@ -1,18 +1,56 @@
-// https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook-trigger?tabs=javascript
+// https://docs.microsoft.com/en-us/azure/azure-functions/
 
-module.exports = async function checkSuiteComplete(context: any, req: any) {
-  context.log('JavaScript HTTP trigger function processed a request.');
+// eslint-disable-next-line node/no-unsupported-features/es-syntax
```

Comment:
This is specifically for the `import` syntax, or something else?

Reply:
Yeah lint was complaining about import